lib/info-extras.js

```diff
@@ -169,3 +169,27 @@ exports.getRelatedVideos = (body) => {
   }
   return videos;
 };
+
+/**
+ * Get like count from html.
+ *
+ * @param {string} body
+ * @return {number}
+ */
+exports.getLikes = (body) => {
+  const regex = /\\"likeCount\\":(\d+?),\\"likeCountText\\"/;
```

Comment:
cache this regex outside of the function so that it's not recompiled each time the function is called. look at `get Author` for example.